src/prashant/Assignment_30/SecondMaxusingArr.java

```diff
@@ -0,0 +1,35 @@
+/*WAP to find Second max from given array
+int[] arr = {10,33,43,55,97,11,3};
+ouput : 55*/
+package prashant.Assignment_30;
+
+public class SecondMaxusingArr {
+	int maxnum = 0;
+	int maxnumArr(int[] arr) {
+			for (int i = 0; i < arr.length; i++) {
+			if (maxnum < arr[i]) {
+				maxnum = arr[i];
+			}
+		}
+		return maxnum;
+	}
+
+	int secondMaxNum(int[] numArr) {
+		int secmaxnum = 0;
+		 maxnum = maxnumArr(numArr);
+		for (int j = 0; j < numArr.length; j++) {
+			if (maxnum != numArr[j]) {
```

Comment:
line 21 and 22 should be in && condition..

if (maxnum < arr[i]) {
				sMax = maxnum;
				maxnum = arr[i];
}else if( maxnum != numArr[j] && numArr[i] > sMax){
				sMax = numArr[i]
}